tracer_test.go

```diff
@@ -348,6 +348,31 @@ var _ = Describe("Tracer", func() {
 			opts.AccessToken = accessToken
 			opts.Tags = tags
 			opts.ConnFactory = fakeConn
+			opts.Recorder = fakeRecorder
+		})
+
+		Context("when the span context is not sampled", func() {
+			It("the span should not be recorded", func() {
+				span := tracer.StartSpan("parent", SetTraceID(1), SetSampled("false"))
+				span.Finish()
+				Expect(fakeRecorder.RecordSpanCallCount()).To(Equal(0))
+			})
+		})
+
+		Context("when the parent span carries the Sampled flag", func() {
+			It("should propagate flags from parent to child", func() {
+				opentracing.SetGlobalTracer(tracer)
+
+				opentracing_parent_span := tracer.StartSpan("parent", SetTraceID(1), SetSampled("false"))
```

Comment:
Please add a test here to ensure that if a span is started with the sampled set to anything else than false, the child span will be recorded.

Reply:
Fixed